src/style.css

```diff
@@ -0,0 +1,22 @@
+
+body {
+  font-family: Arial;
+  font-size: 13px;
+  line-height: normal;
```

Comment:
`line-height: normal` - default value, no need to write it.

Reply:
ok,thanks